src/main/java/gregtech/common/covers/filter/FluidFilterContainer.java

```diff
@@ -54,7 +54,7 @@ public void initUI(int y, Consumer<Widget> widgetGroup) {
         widgetGroup.accept(new LabelWidget(10, y, "cover.pump.fluid_filter.title"));
         widgetGroup.accept(new SlotWidget(filterInventory, 0, 10, y + 15)
             .setBackgroundTexture(GuiTextures.SLOT, GuiTextures.FILTER_SLOT_OVERLAY));
-        this.filterWrapper.initUI(y + 15 + 18 + 5, widgetGroup);
```

Comment:
Why did you move these numbers to WidgetGroupFluidFilter.java?

Reply:
Ì change to have this layout 
![image](https://user-images.githubusercontent.com/9199240/87580731-3b660a00-c6d8-11ea-96b1-5d4aca4dbaba.png)

Reply:
I can see what it does, but I am interested in why did you change it. I think that previous design was better. As it would also allow us to use available space better. We could have 2 columns for modules (filter and #1141 ). With this change there is not much space left on right side and space which we get on bottom is too small for anything. Don't you think? @pyure Your take on this from UX perspective?

Reply:
this doesnt fit anymore.
the new row I added make impossible to add the 3by3 square under the filter
![image](https://user-images.githubusercontent.com/9199240/87639674-e4971980-c745-11ea-941a-57afd67d0ff3.png)

Reply:
Okay then, let's go with current design as it can be adjusted later when needed.